cpp-throughput/throughput_scaling.cpp

```diff
@@ -43,9 +48,7 @@ void run_throughput(std::ofstream& timing_file,
 
     double loop_start = MPI_Wtime();
 
-    // Keys are overwritten in order to help
-    // ensure that the database does not run out of memory
-    // for large messages.
```

Comment:
This may actually be a valid comment to keep. I know if I was auditing the scaling studies this would be something I might want to know why the authors were overriding existing keys in the db instead of creating new ones